tests/mbed/host/host.c

```diff
@@ -31,7 +34,7 @@ char* find_data_file(char* str, size_t size)
         return token;
     }
 
-    strncat(str, tail, strlen(tail));
+    strncat_s(str, TEMP_LEN, tail, strlen(tail));
```

Comment:
Dangerous to hard code length here. Safer to make the caller pass in the size.

Reply:
@ryanhsu19 you should use the `size` variable here